code/modules/reagents/chemistry/holder.dm

```diff
@@ -788,6 +790,42 @@
 	LAZYNULL(failed_but_capable_reactions)
 
 	. = 0
+	for(var/reagent in cached_reagents)
+		var/datum/reagent/R = reagent
+
+		var/turf/T = get_turf(cached_my_atom)
+		switch(R.reagent_state)//evaluating state of matter
+			if(GAS)
+				if(chem_temp < R.boiling_point)
+					R.reagent_state = LIQUID
+					if(!is_type_in_typecache(cached_my_atom, GLOB.no_reagent_message_typecache) && SSticker.HasRoundStarted())
+						for(var/mob/M in viewers(3, T))
+							to_chat(M, ("<span class='notice'>[icon2html(cached_my_atom, viewers(cached_my_atom))] The vapour condenses into a liquid!</span>"))
```

Comment:
use audible message or visible message here you chungus

Reply:
why